NOTICE

```diff
@@ -1,49 +1,309 @@
-NOTICE applicable to Nessie source
-----------------------------------
-
-Dremio
+Nessie
 Copyright 2015-2025 Dremio Corporation
 
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-Apache Avro
-Copyright The Apache Software Foundation
-
-Apache Commons
-Copyright The Apache Software Foundation
-
-Apache Curator
-Copyright The Apache Software Foundation
-
-Apache Hadoop
-Copyright The Apache Software Foundation
-
-Apache HTTP Components
-Copyright The Apache Software Foundation
-
-Apache Iceberg
-Copyright The Apache Software Foundation
-
-Apache Kerby
-Copyright The Apache Software Foundation
+---------------------------------------
+This project includes code from Apache Iceberg, with the following in its NOTICE file:
 
-Apache Polaris (incubating)
-Copyright The Apache Software Foundation
+| Apache Iceberg
```

Comment:
Is it standard to use `|` as the prefix on included lines?

Reply:
It's a way to clearly format the NOTICE file from dependency (to avoid to confuse the reader). It's used in several ASF projects.